18F/identity-idp
# LG-5456: Implement confirmation tooltip for clipboard button

## 🎫 Ticket

[LG-5456](https://cm-jira.usa.gov/browse/LG-5456)

## 🛠 Summary of changes

Adds a new tooltip to be shown when activating "Copy" buttons, using [the U.S. Web Design System Tooltip component](https://designsystem.digital.gov/components/tooltip/) with alterations to display the tooltip only once the button is activated.

## 📜 Testing Plan

1. Go to http://localhost:3000/components/inspect/clipboard_button/preview
2. Click the button
3. Observe the "Copied!" tooltip

## 👀 Screenshots

![Screen Shot 2023-04-28 at 2 45 39 PM](https://user-images.githubusercontent.com/1779930/235229018-43a249ae-6429-4611-98a4-fdb5e735e90b.png)
